.github/workflows/docs.yml

```
@@ -7,13 +7,19 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y pandoc
       - uses: actions/setup-python@v2
         with:
           python-version: '3.8'
           architecture: 'x64'
       # TODO: Dependencies come from environment.yml.
       # How should they be shared here? Install everything?
-      - run: |
-        pip install sphinx==4.2.0 sphinx-rtd-theme==1.0.0 nbclean==0.3.2 nbsphinx==0.8.8
-        pip install -e .
-      - run: make html
\ No newline at end of file
+      - name: Install docs dependencies
+        run: |
+          pip install sphinx==4.2.0 sphinx-rtd-theme==1.0.0 nbclean==0.3.2 nbsphinx==0.8.8 pandoc==2.1
```

I don't _think_ Python pandoc is needed as a dependency

```suggestion
          pip install sphinx==4.2.0 sphinx-rtd-theme==1.0.0 nbclean==0.3.2 nbsphinx==0.8.8
```